cmd/ova-provider-server/ova-provider-server.go

```diff
@@ -382,7 +390,23 @@ func hasSuffixIgnoreCase(fileName, suffix string) bool {
 	return strings.HasSuffix(strings.ToLower(fileName), strings.ToLower(suffix))
 }
 
+func isFileComplete(filePath string) bool {
```

Comment:
```suggestion
// isFileComplete checks that the file was not modified in the last 30s
func isFileComplete(filePath string) bool {
```

just to be clear we don't actually know it's complete, we just have a very good gues

Reply:
@yaacov  - correct , among the other options this is the least time consuming along with the good guess...